.buildkite/nightly.yml

```diff
@@ -61,6 +61,13 @@ steps:
     agents:
       system: x86_64-linux
 
+  - label: 'Purge bench:restore cache'
```

Comment:
This won't work because the pipeline step will run on a random buildkite-agent, and `/cache` is not shared across all agents. See PR #1576 which I just opened.